scripts/pop_demix.py

```diff
@@ -1,22 +1,20 @@
 from neurophotonics.demix import *
+from scripts.helper import keys_used
 
-try:
-    IlluminationCycle.populate(reserve_jobs=True, display_progress=False, processes=1024)
-except ValueError:
-    print("IlluminationCycle already populated")
-
-try:
-    Demix.populate(display_progress=False, reserve_jobs=True, processes=1024)
-except ValueError:
-    print("Demix already populated")
-
-try:
-    Cosine.populate(reserve_jobs=True, display_progress=False, processes=1024)
-except ValueError:
-    print("Cosine already populated")
-
-
-try:
-    SpikeSNR.populate(reserve_jobs=True, display_progress=False, processes=1024)
-except ValueError:
-    print("SpikeSNR already populated")
+print(
+    "keys processed:",
+    keys_used(
+        IlluminationCycle.populate(
```

Comment:
The time-consuming operations are matrix multiplications, which are already parallelized within the BLAS and LAPACK libraries in numpy.  So adding multiprocessing here does not help but may cause memory shortage. Perhaps `processes=2` will yield the highest performance.

Reply:
That seems reasonable. Should I set that for just this table, or all tables in pop_demix?

Reply:
Any `make` call that relies on heavy linear algebra computations in numpy, already makes heavy use of multithreading and does not benefit from additional multiprocessing.

Reply:
That makes sense. The table's multi-processing value has now been set to 2